public/app/features/explore/QueryLibrary/QueryTemplatesList.tsx

```diff
@@ -59,7 +84,11 @@ export function QueryTemplatesList(props: QueryTemplatesListProps) {
             query,
             queryText,
             description: queryTemplate.title,
-            user: queryTemplate.user,
+            user: {
```

Comment:
Not necessary to do here, but would love to have this encapsulated as a react component (in grafana/runtime?) that takes a string input and shows avatar+display name.

Reply:
Yeah but it would be nice, especially for any kind of list view, to not have each component fire it's own request and query if we have API that can batch them.

Reply:
@ryantxu i think that would be nice, but i agree with @aocenas about list views so maybe we should enable the react component to receive multiple uids because that's what the backend API allows

Reply:
I hope this does not do n+1 API requests for each user?

Reply:
@torkelo it doesn't, we batch it here https://github.com/grafana/grafana/blob/19844220db28e55c9c5e61730b6df036bc2b7cf9/public/app/features/explore/QueryLibrary/QueryTemplatesList.tsx#L50. The endpoint accepts multiple uids and responds with a list of users

Reply:
great